src/picosvg/svg_pathops.py

```diff
@@ -183,9 +183,14 @@ def stroke(
 
     # nuke any conics that snuck in (e.g. with stroke-linecap="round")
     sk_path.convertConicsToQuads(tolerance)
-    sk_path.simplify(fix_winding=True)
 
-    assert sk_path.fillType == pathops.FillType.WINDING
+    try:
+        sk_path.simplify(fix_winding=True)
+    except svg_pathops.pathops.PathOpsError:
+        # some tricky paths with very densely packed segments sometimes can trigger a
+        # PathOpsError. We assume they do paint to stay on the safe side.
```

Comment:
The comment you copy pasted doesn't make sense in here, replace or remove at least the last sentence

Reply:
Okay.

Even if an error is reported here, `sk_path.simplify` is still executed, resulting in inconsistent display of pathops-tricky-path-nano.svg.

Reply:
ugh, not good. We need to copy the path before we send to simplify and roll back if an exception occur. I'll merge this and fix up, thanks

Reply:
@yisibl https://github.com/googlefonts/picosvg/commit/727fee175e3523326f15cb02ab4a4e504a8f6e88